### PR TITLE
Enable card present payments on all stores

### LINKED ISSUE #4833: [Card Present Payments] Remove feature flag

### PR DESCRIPTION
**Pull request by malinajirka**
Closes: #4833

### Description
Removes dynamic (api driven) feature flag for in person payments => enables in person payments menu item and collect payment button on all stores.

### Testing instructions

1. Switch to a store which isn't enrolled in the beta
2. Make sure "In Person Payments" menu item is visible in settings
3. Make sure "Collect Payment" button is visible on eligible orders

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 7.7
 -----
-
+- [***] In-Person Payments: Added support for accepting In-Person Card Payments in the US
```
Comment (kidinov) [inline RELEASE-NOTES.txt:5]: 👍

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerPresenter.kt [hunk 2]
```diff
@@ -89,8 +87,6 @@ class SitePickerPresenter
     override fun fetchUserRoleFromAPI(site: SiteModel) {
         coroutineScope.launch {
             val fetchUserJob = async { userEligibilityFetcher.fetchUserInfo() }
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerPresenter.kt]: I think async was added to run 2 requests in parallel. Now probably 
```
val userModel = userEligibilityFetcher.fetchUserInfo()
```
will be enough

File: WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt [hunk 3]
```diff
@@ -18,7 +17,7 @@ enum class FeatureFlag {
                 PackageUtils.isDebugBuild() || context != null && PackageUtils.isBetaBuild(context)
             }
             ORDER_CREATION, WHATS_NEW -> PackageUtils.isDebugBuild() || PackageUtils.isTesting()
-            CARD_READER -> CardPresentEligibleFeatureChecker.isCardPresentEligible
+            CARD_READER -> true
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt]: Maybe to leave a comment to explain why we want to keep the flag for a bit longer?

**Review by kidinov — COMMENTED** [review]
Looks and works well! Just Left a couple of minor comments to think about

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerPresenter.kt (hunk 2 above)
Comment (malinajirka) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerPresenter.kt]: Good call, fixed in 9aad91f831ab5cf37c24cd4526bb54f02be4b0dd

File: WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt (hunk 3 above)
Comment (malinajirka) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt]: Fixed in 9078d4ad9fdb945c5bd747ba5e21ef75a7741f63

**Review by kidinov — APPROVED** [review]
👍

**Pull request merged**